src/adapter/search-request-adapter/__tests__/search-params.tests.ts

```diff
@@ -1,211 +1,194 @@
 import { adaptSearchParams } from '../search-params-adapter'
 
-test('Adapt basic SearchContext ', () => {
-  const searchParams = adaptSearchParams({
-    indexUid: 'test',
-    pagination: { paginationTotalHits: 20, page: 0, hitsPerPage: 6 },
-    defaultFacetDistribution: {},
-    finitePagination: false,
+const DEFAULT_CONTEXT = {
+  indexUid: 'test',
```

Comment:
I think the same idea can be applied here, don't you think? https://github.com/meilisearch/meilisearch-ruby/pull/354/files#r934317471

Reply:
Do you think `test` is not a good index name? Also these are unit tests so they will not be done against meilisearch